inst/templates/derivationchain/body.mustache

```diff
@@ -29,4 +31,5 @@ var statsImage = '{{reportData.derivationChainSymbols.statDerived}}';
 var fillMissingDataImage = '{{reportData.derivationChainSymbols.fillMissingData}}';
 var conditionalDataImage = '{{reportData.derivationChainSymbols.conditionalFill}}';
 var verticalDatumConversionImage = '{{reportData.derivationChainSymbols.verticalDatumConversion}}';
+var noProcessorImage = '{{reportData.derivationChainSymbols.noProcessor}}';
```

Comment:
Well, there was a discussion on how it would be so rare for that icon to ever appear that I kinda just didn't do it, but doesn't hurt to have it I suppose. Either just that it was rare or Aquarius was doing away with it, but again doesn't hurt to have it.